docker-compose.yml

```diff
@@ -112,6 +112,8 @@ services:
       start_period: 10s
 
   payment-service:
+    # build:
+    #   context: ../payment-service
```

Comment:
can just remove these - shouldn't be building here at all

Reply:
[d7a6f5e](https://github.com/ardriveapp/turbo-sdk/pull/196/commits/d7a6f5e8a8aebc717cab30d43c982c04ef422e5b)